Directory.Build.props

```diff
@@ -65,7 +65,7 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <SharedFrameworkName>Microsoft.NETCore.App</SharedFrameworkName>
+    <SharedFrameworkName>Microsoft.WindowsDesktop.App</SharedFrameworkName>
     <NETCoreAppFrameworkIdentifier>.NETCoreApp</NETCoreAppFrameworkIdentifier>
     <NETCoreAppFrameworkMoniker>$(NETCoreAppFrameworkIdentifier),Version=v$(NETCoreAppFrameworkVersion)</NETCoreAppFrameworkMoniker>
     <NETCoreAppFrameworkBrandName>.NET Core $(NETCoreAppFrameworkVersion)</NETCoreAppFrameworkBrandName>
```

Comment:
Might be good to fix these too. I don't know off the top of my head all usages, but at least `NETCoreAppFrameworkIdentifier` is ending up in the framework list, e.g. `microsoft.windowsdesktop.app.ref.5.0.0-preview.3.20214.2.nupkg\data\FrameworkList.xml` first line:

```xml
<FileList Name="Windows Desktop 5.0" TargetFrameworkIdentifier=".NETCoreApp" TargetFrameworkVersion="5.0" FrameworkName="Microsoft.WindowsDesktop.App">
```

Reply:
> ```
> <FileList Name="Windows Desktop 5.0" TargetFrameworkIdentifier=".NETCoreApp" TargetFrameworkVersion="5.0" FrameworkName="Microsoft.WindowsDesktop.App">
> ```

Does this need to change? I see the same in ASP.NET:
```
<FileList Name="ASP.NET Core 3.0" TargetFrameworkIdentifier=".NETCoreApp" TargetFrameworkVersion="3.0" FrameworkName="Microsoft.AspNetCore.App">
```

In both cases FrameworkName is correct.

Reply:
Oh, right, duh, TFM doesn't change per sharedfx. I think maybe the specific prop `NETCoreAppFrameworkBrandName` could be removed (since this repo doesn't depend on the *brand name* of the base shared framework) but it does generally seem good as is.